cli/configssh.go

```diff
@@ -518,6 +531,12 @@ func (r *RootCmd) configSSH() *serpent.Command {
 			Description: "Override the default host prefix.",
 			Value:       serpent.StringOf(&sshConfigOpts.userHostPrefix),
 		},
+		{
+			Flag:        "hostname-suffix",
+			Env:         "CODER_CONFIGSSH_HOST_SUFFIX",
```

Comment:
Nit: Since this is a new flag and the prefix is being deprecated, prefer consistency between flag and env (i.e. `--host-suffix` or `_HOSTNAME_SUFFIX`).